### ISSUE TITLE
CourseGroup API is not working with JWT authentication

### ISSUE THREAD

**Issue by ilanerukh** [ScDor/Closure#123]
**Describe the bug**
Accessing the CourseGroup API works with an admin user, but doesn't work for regular user. However, it should be accessible to all users for GET responses.

**To Reproduce**
1. Login to the site, and get your idToken
2. Go to CourseGroup api, and you will be unauthorized.

**Expected behavior**
The user should have an access to this API.

**Screenshots**

**Additional context**
Basically it's just changing the permission class in the View. will do.
